go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/mitchellh/go-ps v1.0.0
 	github.com/moricho/tparallel v0.2.1
 	github.com/nakabonne/nestif v0.3.0
-	github.com/nishanths/exhaustive v0.0.0-20200811152831-6cf413ae40e0
+	github.com/nishanths/exhaustive v0.0.0-20201012211509-7221e29c71f4
```

Comment:
as you are maintainer of exhaustive, can you release tag version vX.Y.Z ? so that going forward dependenbot will just update it automatically ?

Reply:
I am. I'll do that.